rsrcdump/pict.py

```diff
@@ -275,7 +279,7 @@ def read_pict_bits(u: Unpacker, opcode):
             mask = unpack_maskrgn(maskrgn_bits, mask_w, mask_h)
             #print(binascii.hexlify(maskrgn_bits, ' ', 2))
 
-    if opcode in [0x0091, 0x009b]:
+    if opcode in [0x0091, 0x009b] or palette is None:
```

Comment:
```suggestion
    if opcode in [0x0091, 0x009b]:
```
There's no palette with opcode 009A (RGB pixmap)